indico/modules/auth/templates/emails/create_local_identity.txt

```diff
@@ -0,0 +1,18 @@
+{% extends 'emails/base_i18n.txt' %}
+
+{% block subject -%}
+    {% trans %}Create local account{% endtrans %}
+{%- endblock %}
+
+{% block header_recipient -%}
+    {{ user.first_name }}
+{%- endblock %}
+
+{% block body -%}
+{% trans %}We heard that you forgot your Indico username or password.{% endtrans %}
```

Comment:
Should we mention username here or just password? The link says `Reset password` and there's no mention of forgotten username anywhere besides here, so it could be a bit confusing perhaps?